registration-processor-default.properties

```diff
@@ -478,13 +478,22 @@ mosip.biometric.sdk.providers.face.mosip-ref-impl-sdk-client.format.url.mock-1.1
 mosip.registration.processor.credentialtype=euin
 mosip.registration.processor.encrypt=false
 mosip.registration.processor.credential.request.service.id=mosip.credential.request.generator 
-mosip.registration.processor.digitalcard.credentialtype=PDFCard
-mosip.registration.processor.print.credentialtype=euin
-mosip.registration.processor.digitalcard.template=RPR_UIN_CARD_TEMPLATE
-mosip.registration.processor.print.template=RPR_UIN_CARD_TEMPLATE
-
-# this property is used to define issuers eg partnerid:credentialType:templateTypeCode
-mosip.registration.processor.issuer=mpartner-default-print:${mosip.registration.processor.print.credentialtype}:${mosip.registration.processor.print.template};mpartner-default-digitalcard:${mosip.registration.processor.digitalcard.credentialtype}:${mosip.registration.processor.digitalcard.template}
+# This property is used to define default internal credential issuers separated by comma.
+# eg: partnerid#credentialType#templateTypeCode
+mosip.registration.processor.default.internal.issuers=mpartner-default-digitalcard#PDFCard#RPR_UIN_CARD_TEMPLATE
+# This property is used to define default issuers separated by comma,
+# eg: partnerid#credentialType#templateTypeCode
+mosip.registration.processor.default.issuers=mpartner-default-print#euin#RPR_UIN_CARD_TEMPLATE
+
+# Credential Partner Configuration
+# First language defined in the mandatory language list will be considered to identify the attribute value.
+mosip.mandatory-languages=eng
+# Map all print partners for a respective attribute ('state') value (any appropriate valid id-schema field) in the format given below,
+# key and value pair where key would be string value (partnerId#credential_type#template_code) and value would be an expression
+# ({list_of_attribute_values} contains attribute_field)
+mosip.registration.processor.print.issuer.config-map={'mpartner-msp-print#vercred#RPR_UIN_CARD_TEMPLATE':'{"Karnataka", "Tamil Nadu"} contains ${mosip.identification.attribute}', 'mpartner-pdf-print#vercred#RPR_UIN_CARD_TEMPLATE': '{"Andra Pradesh"} contains ${mosip.identification.attribute}'}
```

Comment:
Partners mentioned in this config mpartner-msp-print and mpartner-pdf-print are not part of default partners in the system, this will cause issue when the credential is processed. So we can have them as example in comments and use the mpartner-default-print in the default sample

Reply:
yes, changed accordingly.